src/import/FileInfo.ts

```diff
@@ -0,0 +1,3 @@
+export class FileInfo {
```

Comment:
So... thinking about what this represents -- and the fact that it might not actually come from a file (for example, in our unit tests, but also if we support a web-based IDE) -- the name `FileInfo` didn't seem _quite_ right.  But I was having trouble coming up with something better.

Then it hit me.

...

`RawFSH`.  How do you feel about renaming this to `RawFSH`?